### ISSUE TITLE
missing file "predation-accessibility.csv” 

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/fb-osmose-bridge#7]
as observed by Arnaud Gruss:
"The “predation-accessibility.csv” file is missing in “osmose_config.zip”"

**Comment by jhpoelen** [comment]
missing files has been added. closing issue.
